### ISSUE TITLE
Make climatological days configurable

### ISSUE THREAD

**Issue by aryiu** [Daveiano/weewx-wdc#265]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like to be able to add more types of climatological days in Statistics page (like the existing `tropicalNights`). These concepts are very common, for example, in the Iberian Peninsula in Summer:

- `equatorialNights`: min. temperature is above or equal to 25 °C.
- `hellishNights`: min. temperature is above or equal to 30 °C.

**Describe alternatives you've considered**
I tried to add them manually with no success.

**Additional context**
Current climatological days table:

<img width="600" alt="climdays" src="https://github.com/Daveiano/weewx-wdc/assets/130826840/31d7d84c-61e7-4cf8-8af1-bf1818d32786">

**Comment by Daveiano** [comment]
Yes, it would be great if all these climatological days could be configured by the user. This way new ones could be added, and days which don't make sense in some places, could be removed. 

Thank you again for all the feedback, I will try to include the most in the next release!

**Comment by aryiu** [comment]
Exactly, completely agree. It would be awesome. Thanks!
